src/util/std_types.cpp

```diff
@@ -15,6 +15,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "string2int.h"
 #include "arith_tools.h"
 #include "std_expr.h"
+#include "namespace.h"
```

Comment:
As above: please sort lexicographically, likely making a slightly bigger change than necessary (sort all of them).